ixofoundation/ixo-webclient
# LaunchPad Voting Campaign

## Scope
[Ticket: WEB-270](https://ixo.atlassian.net/browse/WEB-270)

## Work Done
- LaunchPad Card as well as on Figma
- Add Stake to Vote button on Actions Panel when  launchPad is selected.
- Change HeaderTab from FUNDING to VOTING when launchPad is selected.
- And this VOTING takes to Voting Bond Container
- Voting Bond Headline indicators with specified endpoints.
- Voting Bond history chart with bondDid(this is same as Bond interface)
- add `Stake to Vote` in **config.json**

## Steps to Test
- In Entities Explorer, Filter Sector as `Campaign` and Stage as `Selection` and Project Type as `Candidate` to see LaunchPad Card
- Click on Card will take to Overview scene
- Here you can see VOTING tab item and Stake to Vote button on Actions panel
- Click Voting tab item to redirect Voting Bond Container

## Screenshots
[Loom link](https://www.loom.com/share/998f9e83a5e34a6f9c473602ba3a9c2b)
note: In Voting Bond interface, bond hasn't transaction history, so it displays nothing.
